index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    
+</head>
+<body>
+    <header class="header">
+        <div class="header-contain">
+            <div class="logo">My Logo</div>
+            <div>
+                <img class="vector" src="./assets/img/Union.png" alt="">
+                
+            </div>
+        </div>
+    </header>
+    <section>
+        <div class="main__section">
+            <div>
```

Comment:
- [ ] `main-section` content should be aligned with [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/). This gives us a lot of options to align all our elements, plus, it's responsive. Kindly ensure that you use `display: flex` on this part. ☺️